seleniumByTechnocredits/src/seleniumPractise/purva/Assignment1_5.java

```diff
@@ -0,0 +1,110 @@
+//below program is for validation alert text when password is greater than or equal to 8 characters
+
+package seleniumPractise.purva;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import util.Init;
+
+public class Assignment1_5 {
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		driver.get("http://automationbykrishna.com/#");
+		Thread.sleep(1000);
+		// clicked on registration page
+		driver.findElement(By.xpath("//a[@id='registration2']")).click();
+		Thread.sleep(1000);
+		System.out.println("This is alert password program");
+
+		// sending Purva to username field
+		driver.findElement(By.xpath("//input[@id='unameSignin']")).sendKeys("Purva");
+		Thread.sleep(2000);
+		// 1st input for password is blank
+		String password0 = "";
+		WebElement element = driver.findElement(By.xpath("//input[@id='pwdSignin']"));
+		element.sendKeys(password0);
+		Thread.sleep(1000);
+		// driver.findElement(By.xpath("//button[@id='btnsubmitdetails']"));
+		driver.findElement(By.xpath("//button[@id='btnsubmitdetails']")).click();
+		Thread.sleep(2000);
+		Alert alert0 = driver.switchTo().alert();
+		String actualtextp0 = alert0.getText();
+		String expectedtexp0 = "Failed! please enter password";
+		if (password0.length() == 0) {
```

Comment:
if statement can be avoided as length of password is handled internally. You can just validate alert by giving password of various length.